llm: Add commands for system prompt and llm update (#12624)
tldr-pages/tldr: pages/common/llm.md

`llm "{{Ten fun names for a pet pelican}}"`

> - Run a system prompt against a file:

Reviewer: ```suggestion
- Run a [s]ystem prompt against a file:
```

`cat {{path/to/file.py}} | llm --system "{{Explain this code}}"`

- Install packages from PyPI into the same environment as LLM:

`llm install {{package1 package2 ...}}`

> - Download and run a prompt against a model:

Reviewer: ```suggestion
- Download and run a prompt against a [m]odel:
```

`llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`

> - Create a system prompt:

Reviewer: ```suggestion
- Create a [s]ystem prompt and save it with a template name:
```

Author: Shouldn't you also put square brackets to [t]emplate since it also highlights short options for the example `{{sentient_cheesecakes}}`?

Author: Also, I didn't know about using square brackets to highlight short options, and I also couldn't find it on the [contribution guideline](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md). It's probably better to include it in the contribution guideline as well, because there are quite a few commands that don't follow the highlighting rule for short options.

Reviewer: > Shouldn't you also put square brackets to [t]emplate since it also highlights short options for the example `{{sentient_cheesecakes}}`?

Yes, updated the suggestion.

> Also, I didn't know about using square brackets to highlight short options, and I also couldn't find it on the [contribution guideline](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md). It's probably better to include it in the contribution guideline as well, because there are quite a few commands that don't follow the highlighting rule for short options.

We have documented this in detail in the style guide at <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md>, the `CONTRIBUTING.md` file is a quick reference to get started, whereas the style guide (referenced in the CONTRIBUTING file too) contains all the nuances required.

I will try to add a note at the top of the CONTRIBUTING file clarifying this and instead link to the style guide for extended syntax.

> `llm -s 'You are a sentient cheesecake' --save {{sentient_cheesecake}}`

Reviewer: ```suggestion
`llm --system '{{You are a sentient cheesecake}}' --save {{sentient_cheesecake}}`
```

> - Have an interactive chat with a specific model using a specific template:

Reviewer: ```suggestion
- Have an interactive chat with a specific [m]odel using a specific [t]emplate:
```

`llm chat --model {{chatgpt}} --template {{sentient_cheesecake}}`

> 
> - Update llm:
> 
> `pip install -U llm`

Reviewer: ```suggestion
```
We don't document global package manager commands (i.e. `pip` in this case) to download/update specific packages. So, I would suggest reverting it.

`llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`

> - Create a [s]ystem prompt and save it with a [t]emplate name:

Author: Thanks!
